site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -238,19 +238,29 @@ public class MyAnnotatedService {
 When the value of <type://@Param> annotation is not shown in the path pattern, it will be handled as a
 parameter name of the query string of the request. If you have a service class like the example below and
 a user sends an HTTP `GET` request with URI of `/hello1?name=armeria`, the service method will get `armeria`
-as the value of parameter `name`. If there is no parameter named `name` in the query string, the parameter
-`name` of the method would be `null`. If you want to avoid `null` in this case, you can use
-<type://@Default> annotation or `Optional<?>` class, e.g. `hello2` and `hello3` methods below, respectively.
+as the value of parameter `name`.
+
+If there is no parameter named `name` in the query string, the service method that requires it will not be
+invoked, but the client will get a `400 Bad Request` response.  If you want to allow `null` to be injected,
+you can use <type://@Default> or `@Nullable` annotation or `Optional<?>` class, like `hello2()`, `hello3()`
```

Comment:
`<type://@Default>, @Nullable annotation or Optional<?> class`?

Reply:
I meant (Default or Nullable) annotation or Optional class. What would be better, @KarboniteKream ?

Reply:
Ah, I missed that. 😄

Reply:
Maybe something like `you can use the @Default or @Nullable annotations or an Optional<?> class`. A comma after `annotation` might also work.

Reply:
Changed to:

```md
you can use <type://@Default> annotation, `@Nullable` annotation or `Optional<?>` class, like demonstrated
below in `hello2()`, `hello3()` and `hello4()` methods:
```